nemo/core/neural_types/__init__.py

```diff
@@ -19,3 +19,4 @@
 from nemo.core.neural_types.comparison import *
 from nemo.core.neural_types.elements import *
 from nemo.core.neural_types.neural_type import *
+from nemo.core.neural_types.nmtensor_registry import NmTensorNameRegistry
```

Comment:
Analogical comment that I got from @okuchaiev regarding the other registries: do we want the user to be able to access it?

If not, please remove it from the namespace.

Reply:
I thought this should be visible for users who write their own callbacks. Did I get it wrong?

Reply:
Removed